tsconfig.node.json

```diff
@@ -11,11 +11,12 @@
                 "./src/*",
             ]
         },
+        "types": ["vite/client", "vitest/globals"]
     },
     "include": [
         "vite.config.ts",
         "vite-env.d.ts",
         "vitest.config.ts",
         "playwright.config.ts"
     ]
-}
\ No newline at end of file
+}
```

Comment:
Please confirm that you have the .editorconfig plugin installed for your editor: https://editorconfig.org/

(I'll be sure to add this to the readme to make it clearer)

Reply:
Oh I might be looking at this backwards... looks like you are following the correct formatting